services/app/src/pages/home/HomePage.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from 'react';
+import { useNavigate } from "react-router-dom";
+import { useGameIdFromLocalStorage } from '../../hooks/useGameIdFromLocalStorage';
+
+function HomePage() {
+  const navigate = useNavigate();
+
+  const gameId = useGameIdFromLocalStorage();
+
+  useEffect(() => {
+    if(gameId) navigate(`/game/${gameId}`);
+  })

```

Comment:
You should avoid using `useEffect` without dependencies array, as that would cause the effect to run on every render which is almost never what you want.

An empty dependencies array `[]` should do the trick here, as it would only invoke the effect on component mount.